Automattic/crowdsignal-ui
# Add multiple choice support for the Dropdown block

## Summary

This PR adds a toolbar option to switch between `Single Choice` and `Multiple Choices` for the Dropdown block.

![image](https://user-images.githubusercontent.com/7811225/175335332-3ea7c680-927e-4e9b-97f5-1480f2ba86ec.png)


## Testing Instructions

* Create/edit a project and add a Dropdown block
* Check if the Toolbar shows the Multiple/Single choice option
* Go to the Renderer and check if the block works as expected
